examples/metronome/main.go

```diff
@@ -43,58 +27,86 @@ func main() {
+		Enabled:                 true,
+		StartingDeadlineSeconds: 60,
+		Timezone:                "America/Chicago",
 	}
-	// Get Job
-	retJob, _, err := client.Metronome.V1GetJob(ctx, jobID, nil)
-	if err != nil {
-		log.Fatal(err)
+	mjs, resp1, err2 := client.Metronome.V1CreateJobSchedules(ctx, jobID, v1mcjs)
+	if err2 != nil {
+		log.Println(err2)
 	}
-	fmt.Printf("Job with ID: %s found - %#v", jobID, retJob)
+	log.Printf("V1CreateJobSchedules object: %+v\n", mjs)
+	log.Printf("V1CreateJobSchedules Status code: %s\n", resp1.Status)
 
-	updateJobDescription := "Updated jobs description"
-	// Update Job
-	retJob.Description = updateJobDescription
-	_, _, err = client.Metronome.V1UpdateJob(ctx, jobID, retJob)
-	if err != nil {
-		fmt.Printf("Error: update did not work %v\n", err)
-	} else {
-		fmt.Printf("Updated job with id %s\n", jobID)
+	// Get job schedule
+	mv1js, resp3, err3 := client.Metronome.V1GetJobSchedules(ctx, jobID)
+	if err3 != nil {
+		log.Println(err3)
+	}
+	log.Printf("V1GetJobSchedules: %+v\n", mv1js)
+	log.Printf("V1GetJobSchedules Status code: %s\n", resp3.Status)
 
+	// Start a run
+	mv1job2, resp4, err4 := client.Metronome.V1StartJobRun(ctx, jobID)
+	if err4 != nil {
+		log.Println(err4)
 	}
+	log.Printf("V1StartJobRun object: %+v\n", mv1job2)
+	log.Printf("V1StartJobRun Status code: %s\n", resp4.Status)
 
-	// Get job again and compare Description
-	retupdatedJob, _, err := client.Metronome.V1GetJob(ctx, jobID, nil)
-	if err != nil {
-		log.Fatal(err)
+	time.Sleep(5 * time.Second)
+
+	// Get job by run ID
+	mv1job3, resp5, err6 := client.Metronome.V1GetJobRunByRunId(ctx, jobID, mv1job2.Id)
+	if err6 != nil {
+		log.Println(err6)
 	}
+	log.Printf("V1GetJobRunByRunId: %+v\n", mv1job3)
+	log.Printf("V1GetJobRunByRunId Status code: %s\n", resp5.Status)
 
-	if retupdatedJob.Description != updateJobDescription {
-		fmt.Printf("Error: update did not work\n")
-	} else {
-		fmt.Printf("Update successfull\n")
+	// Stop job by run ID
+	resp6, e := client.Metronome.V1StopJobRunByRunId(ctx, jobID, mv1job2.Id)
+	if e != nil {
+		log.Println(e)
 	}
+	log.Printf("V1StopJobRunByRunId Status code: %s\n", resp6.Status)
 
-	resp, err = client.Metronome.V1DeleteJob(ctx, jobID)
-	if err != nil {
-		log.Fatal(err)
+	time.Sleep(5 * time.Second)
+
+	// Get Schedule
+	mv1js2, resp7, err9 := client.Metronome.V1GetJobSchedulesByScheduleId(ctx, metronomeV1Job.Id, v1mcjs.Id)
+	if err9 != nil {
+		log.Println(err9)
 	}
+	log.Printf("V1GetJobSchedulesByScheduleId object: %+v\n", mv1js2)
+	log.Printf("V1GetJobSchedulesByScheduleId Status code: %s\n", resp7.Status)
 
-	fmt.Printf("Deleted Job %s\n", resp.Status)
+	// Delete schedule
+	resp7, err8 := client.Metronome.V1DeleteJobSchedulesByScheduleId(ctx, metronomeV1Job.Id, v1mcjs.Id)
+	if err8 != nil {
+		log.Println(err8)
+	}
+	log.Printf("V1DeleteJobSchedulesByScheduleId Status code: %s\n", resp7.Status)
 
+	// Delete job
+	resp5, err5 := client.Metronome.V1DeleteJob(ctx, jobID)
```

Comment:
nit: it reads better to re-use the previous variables

Reply:
I did new vars to not shadow the previous ones.

Reply:
@bamarni do you want me to change, or? I did new variables because of the variable "shadowing" annoyances in go.

Reply:
Yes I think it reads better to stick to `resp` and `err`, there is no need to re-use the previous vars afterwards so it seems ok to overwrite them.

Reply:
@hehnope i also vote for readability.

In general all this is in the same block so there shouldn't be any shadowing

Reply:
Fixed in af6c1571ced660ee6b0dc1a1541253875a7f06b5